components/header-bar/src/profile-menu/instance-and-app-info.js

```diff
@@ -0,0 +1,78 @@
+import { MenuItem } from '@dhis2-ui/menu'
+import { useAlert, useConfig } from '@dhis2/app-runtime'
+import { colors } from '@dhis2/ui-constants'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+const useDebugInfo = () => {
+    const { appName, appVersion, systemInfo } = useConfig()
+
+    return {
+        dhis2_version: systemInfo?.version || 'unknown',
+        dhis2_revision: systemInfo?.revision || 'unknown',
+        app_name: appName || 'App',
+        app_version: appVersion?.full || 'unknown',
```

Comment:
Do we need to wrap the "unknown"s with `i18n.t(...)`?

Reply:
I don't think so, as this is debug information that's intended for administrators / our Jira

Reply:
`unknown` should also only ever appear if the headerbar component is used in an older app-platform version.  We could perhaps use an empty string instead, but I feel like that would be confusing...

Reply:
Or we could hide the menu item in that case.

Reply:
cc @cooper-joe

Reply:
Could we use `Couldn't get DHIS2 version` and `Couldn't get app version` instead of `unknown`?

Reply:
> Could we use Couldn't get DHIS2 version and Couldn't get app version instead of unknown? Though it's an edge case, would be good to let the user know what's happening.

I think it wouldn't hurt us to translate these. For us, it's just wrapping the string with `i18n.t('....')`

Reply:
Maybe put this in the menu item (and translate it):

```
Unknown DHIS2 version
Unknown app version
```
?

Reply:
I'd advocate for not translating the details in the modal, or anything that's copied to clipboard

Reply:
We could maybe translate the keys in the table... but as we add more and more information there it could get complicated

Reply:
> Maybe put this in the menu item (and translate it):
> 
> ```
> Unknown DHIS2 version
> Unknown app version
> ```
> 
> ?

Sounds good to me 👍